src/main/java/com/example/bankservicedemo/service/impl/TransactionServiceImpl.java

```diff
@@ -0,0 +1,61 @@
+package com.example.bankservicedemo.service.impl;
+
+import com.example.bankservicedemo.exception.NoEnoughMoneyException;
+import com.example.bankservicedemo.model.Account;
+import com.example.bankservicedemo.model.Transaction;
+import com.example.bankservicedemo.repository.TransactionRepository;
+import com.example.bankservicedemo.service.AccountService;
+import com.example.bankservicedemo.service.TransactionService;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class TransactionServiceImpl implements TransactionService {
+    private final TransactionRepository transactionRepository;
+    private final AccountService accountService;
+
+    @Override
+    public void transfer(String fromAccount, String toAccount, int amount) {
```

Comment:
```suggestion
    public void transfer(Account fromAccount, Account toAccount, int amount) {
```
and maybe we could move this to AccountService - how do you think?